src/data/battle_moves.h

```diff
@@ -13513,8 +13513,8 @@ const struct BattleMove gBattleMoves[MOVES_COUNT_DYNAMAX] =
         .priority = 0,
         .split = SPLIT_SPECIAL,
         .zMoveEffect = Z_EFFECT_NONE,
-        .metronomeBanned = TRUE, // Only since it isn't implemented yet
         .forcePressure = TRUE,
+        .metronomeBanned = TRUE, // Only since it isn't implemented yet
```

Comment:
I know this isn't actually new, but [`EFFECT_PLACEHOLDER` moves can't be selected by Metronome](<https://github.com/rh-hideout/pokeemerald-expansion/blob/master/src/battle_script_commands.c#L12444-L12448>), so it's unnecessary (and harmful, because somebody might forget to remove that line).

Reply:
I did not know this, thanks for telling me.